### DISCUSSION TITLE
[Bug]: OpenAI Azure Endpoint not showing

### DISCUSSION THREAD

**Discussion by thunderbug1** [danny-avila/LibreChat#703] [Troubleshooting] [answered]
### What happened?

I am using LibreChat@0.5.4 backend from ghcr.io/danny-avila/librechat:latest

I configured the following settings but the Azure OpenAI endpoint is not selectable :

    AZURE_OPENAI_API_KEY           = "********"
    AZURE_OPENAI_API_INSTANCE_NAME = "myendpoint"
    AZURE_OPENAI_MODELS                          = "gpt-35-turbo" #,gpt-4"
    AZURE_OPENAI_API_DEPLOYMENT_NAME             = "chat"
    AZURE_OPENAI_API_VERSION                     = "2023-05-15"
    AZURE_OPENAI_API_COMPLETIONS_DEPLOYMENT_NAME = "chat"
    AZURE_OPENAI_API_EMBEDDINGS_DEPLOYMENT_NAME  = "embedding-gen"
    PLUGINS_USE_AZURE="true"

### Steps to Reproduce

1. start the docker container with the settings of an Azure openai deployment
2. try to chat with it

### What browsers are you seeing the problem on?

Chrome

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/1395321/e2cabf4d-dc7d-4563-a760-1a66392ef02a)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
I had to change the environment variable from AZURE_OPENAI_API_KEY to AZURE_API_KEY, because the former would be read by langchain and cause issues when a user has both Azure and OpenAI keys set. This is a known issue in the langchain library https://github.com/hwchase17/langchainjs/issues/1687

**Reply by danny-avila** [reply]
Sorry this was not properly documented; though I had changed .env.example, I have now added it here as well: https://docs.librechat.ai/general_info/breaking_changes.html#v053
